### PR TITLE
Feat(Manager): Add dist sort on manager.

### PR DESCRIPTION
**Pull request by nkokla**

### CODE REVIEW

File: src/components/Manager/ManagerHeader.tsx [hunk 1]
```diff
@@ -103,14 +103,14 @@ const ManagerHeader = ({
...
 
-    setFilteredDemands(filteredDemands);
+    onFilterUpdate(filteredDemands);
   }, [
     addressFilter,
     nameFilter,
     filterModeChauffage,
     filterTypeChauffage,
     demands,
-    setFilteredDemands,
```
Comment (desoindx) [inline src/components/Manager/ManagerHeader.tsx:113]: I find this change pretty aggressive as i explicitely refused to do it :/

File: src/components/Manager/Manager.tsx [hunk 2]
```diff
@@ -19,27 +20,76 @@ import ManagerHeader from './ManagerHeader';
 import Status from './Status';
 import Tag from './Tag';
 
+type SortParamType = {
+  key?: keyof Demand;
+  order?: 'asc' | 'desc';
+};
+
+const getSortBy =
```
Comment (desoindx) [inline src/components/Manager/Manager.tsx:28]: there is already a sortBy implemented on the react dsfr

File: src/types/Summary/Demand.d.ts [hunk 3]
```diff
@@ -16,9 +16,10 @@ export type EditableDemandSummary = {
 export type Demand = DemandSummary &
   EditableDemandSummary & {
     Mail: string;
-    'Distance au réseau': string;
+    'Distance au réseau': number;
     'N° de dossier': string;
     'en ZDP': string;
+    'Date de la demande': string;
```
Comment (desoindx) [inline src/types/Summary/Demand.d.ts:22]: We should not use `Date de la demande` but always `Date demandes`

**Review by desoindx — CHANGES_REQUESTED** [review]

File: src/types/Summary/Demand.d.ts (hunk 3 above)
Comment (nkokla) [reply src/types/Summary/Demand.d.ts:22]: I think so too. Maybe we should merge these two columns in our data sources because is it some confusing. Also, i fix that.

File: src/components/Manager/ManagerHeader.tsx (hunk 1 above)
Comment (nkokla) [reply src/components/Manager/ManagerHeader.tsx:113]: I'm sorry you're seeing an aggressive update because that's absolutely not the goal. And also, this is not for the purpose for opposite the code you had provided. 
In this feature I am passing a complete function to call after update the filter and not just an update status so in my mine this is more appropriate. (And when i have submit this name, in your PR, it is in the projection of this)
On this line, it's just the name of the function in the props. If you prefer, I can go back to using your original "setFilteredDemands" name.

File: src/components/Manager/Manager.tsx [hunk 4]
```diff
@@ -67,8 +119,16 @@ const Manager = () => {
       render: (demand) => <Addresse demand={demand} />,
     },
     {
-      name: 'Date de la demande',
-      label: 'Date de demande',
+      name: 'Date demandes',
```
Comment (desoindx) [inline src/components/Manager/Manager.tsx:122]: Why did you change this label ?

**Review by desoindx — APPROVED** [review]
